src/InputObserver.ts

```diff
@@ -90,13 +99,17 @@ export class InputObserver implements InputTypeObserver {
     }
     depaPos = this._atOutside(depaPos);
     destPos = this._atOutside(destPos);
+
+    if (!this.options.nested || !this._isEndofAxis(offset, depaPos, destPos)) {
+      nativeEvent.childrenAxesAlreadyChanged = true;
```

Comment:
This behavior exists for multiple Axes handling a single pointermove browser event.

Axes does not check AxesEvents occurred by other Axes instances. This is a way to notify other Axes instances that a valid coordinate change has already occurred in another Axes. Which can possible by checking the native event triggering change.

Reply:
ok. 
I understood using events as a way to share data between multiple axes.

Reply:
It can be sensitive because it uses nativeEvent. I'd like to fix the property names. What about names with underscores?

`nativeEvent.__childrenAxesAlreadyChanged`